pkg/daemon/ceph/osd/volume.go

```diff
@@ -127,9 +146,29 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 				"--data",
 				deviceArg,
 			}...)
+
 			// execute ceph-volume with the device
+			op, err := context.Executor.ExecuteCommandWithCombinedOutput(false, "", baseCommand, immediateExecuteArgs...)
+			if err != nil {
+				cvLogFilePath := path.Join(cvLogDir, "ceph-volume.log")
+
+				// Open c-v log file
+				cvLogFile, err := os.Open(cvLogFilePath)
+				if err != nil {
+					logger.Errorf("failed to open ceph-volume log file %q. %v", cvLogFilePath, err)
+				}
+				defer cvLogFile.Close()
+
+				// Read c-v log file
+				b, err := ioutil.ReadAll(cvLogFile)
+				if err != nil {
+					logger.Errorf("failed to read ceph-volume log file %q. %v", cvLogFilePath, err)
+				}
+
+				// Print c-v log before exiting
+				logger.Errorf("%s", string(b))
```

Comment:
should this be in an `else`?

Reply:
Well we are in a failure condition already if anything fails before that we log it and this will print empty.